Cargo.toml

```diff
@@ -38,6 +39,8 @@ xxx_debug_only_print_generated_code = ["wasm-bindgen-macro/xxx_debug_only_print_
 wasm-bindgen-macro = { path = "crates/macro", version = "=0.2.48" }
 serde = { version = "1.0", optional = true }
 serde_json = { version = "1.0", optional = true }
+uluru = "0.3.0"
```

Comment:
This'll want to be an optional dep I think, right?

Reply:
Ah, good catch!